src/app/app.routes.ts

```diff
@@ -73,6 +73,7 @@ export const APP_ROUTES: Routes = [
   {
     path: '',
     component: AppLayoutComponent,
+    canActivate: [AuthGuard],
```

Comment:
can we keep both guards in the same place?

Reply:
I don't think is needed, the AuthGuard should be smart enough to check if a user is logged in SSO/BPM/ECM